analytics_data_api/v0/urls/course_summaries.py

```diff
@@ -4,6 +4,8 @@
 
 from analytics_data_api.v0.views import course_summaries as views
 
+app_name = 'analytics_data_api.v0.course_summaries'
```

Comment:
Note that this is only used as an optional prefix when looking up URLs by name.  We haven't used the ones previously declared in the `include()` calls anywhere, so to some extent it doesn't matter what the value is.  But these are long enough that I'm not sure we'd ever want to use them; `reverse("analytics_data_api.v0.course_summaries:course_summaries")` is a little verbose.

Maybe we'd just be fine using values like `app_name = 'course_summaries'`, since that's effectively what we had before?